media/rendering.html

```diff
@@ -18,55 +18,61 @@ <h2>Well-deployed technologies</h2>
         </div>
         <div data-feature="Distributed rendering">
           <p>The Multi-Device Timing Community Group is exploring another aspect of multi-device media rendering: its <a data-featureid="timing">Timing Object</a> specification enables to keep video, audio and other data streams in close synchrony, across devices and independently of the network topology. This effort needs support from interested parties to progress.</p>
-          <p>To improve the interoperability of implementations of the Presentation API and Remote Playback API, in particular between the first and second screen, the Second Screen Community Group is discussing requirements for an <a data-featureid="secondscreen-openscreen">Open Screen Protocol</a>.</p>
-        </div>
-        <div data-feature="Rendering in different color spaces">
-          <p>To adapt to wide-gamut displays, all the graphical systems of the Web will need to adapt to these broader color spaces. <a data-featureid="css-color-space/icc-colors">CSS Colors Level 4</a> is proposing to define CSS colors in color spaces beyond the classical sRGB. Similarly, work on <a data-featureid="color-canvas">making canvas color-managed</a> should enhance the support for colors in HTML Canvas.</p>
         </div>
       </section>
       <section>
         <h2>Features not covered by ongoing work</h2>
         <dl>
           <dt>Color Management</dt>
-          <dd>To ensure the proper rendering of videos with high-dynamic range (HDR) and wide-gamut colors, content providers would need to determine whether the underlying device and browser have proper support for this. Similarly, content providers need a mechanism to match colors to mix HDR content and Standard Dynamic Range (SDR) content. The <a href="https://www.w3.org/community/colorweb/">Color on the Web Community Group</a> allows color experts from various fields to share ideas and discuss technical solutions to improve the state of Color on the Web.</dd>
+          <dd>To ensure the proper rendering of videos with high-dynamic range (HDR), on top of means to target color spaces and determine whether the underlying device and browser support them (respectively addressed in <a data-featureid="css-color-space/icc-colors">CSS Colors</a> and <a data-featureid="mediaqueries5">Media Queries</a>), content providers also need a mechanism to match colors to mix HDR content and Standard Dynamic Range (SDR) content. The <a href="https://www.w3.org/community/colorweb/">Color on the Web Community Group</a> allows color experts from various fields to share ideas and discuss technical solutions to improve the state of Color on the Web.</dd>
 
           <dt>Native support for 360° video rendering</dt>
```

Comment:
Should we add caption rendering in VR/360 experience, either here or under Exploratory work?